Dockerfile.ci

```diff
@@ -1,10 +1,11 @@
 ARG commit=develop
-FROM elifesciences/proofreader-php:0.1 AS proofreader
+FROM elifesciences/proofreader-php:0.2 AS proofreader
 FROM elifesciences/annotations_cli:${commit}
 ENV DEBUG=1
 
 USER root
-RUN mkdir -p build && chown www-data:www-data build
+RUN mkdir -p build && chown www-data:www-data build && \
+    touch .php_cs.cache && chown www-data:www-data .php_cs.cache
```

Comment:
this is a default, but it can be overridden with a volume if needed

Reply:
(so I'll separate that in a follow-up PR as there may be some decisions to take like where to put it on the host and in the container)